src/main/java/com/github/fge/jsonpatch/PathValueOperation.java

```diff
@@ -73,6 +73,10 @@ public final void serializeWithType(final JsonGenerator jgen,
         serialize(jgen, provider);
     }
 
+    public JsonNode getValue() {
+        return value;
```

Comment:
This is the one where if `JsonNode` is mutable, then callers can change the value out from under the `Operation`. Should we not protectively `deepCopy` here?

Reply:
Added `deepCopy` as suggested, I didn't realise `JsonNode` is mutable so good spot!